.travis.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - stage: Lint
       name: whitespace
       os: linux
-      script: "[[ -z \"$(git ls-files | xargs egrep -l '\\s+$')\" ]]"
+      script: "[[ -z \"$(git ls-files | grep -v '.png$' | xargs egrep -l '\\s+$')\" ]]"
```

Comment:
```suggestion
      script: "git --no-pager grep --color -line-number --full-name -I '\\s+$'"
```
@ktbarrett, thoughts?

Reply:
When trying this locally, I need to escape the ``+`` too - what works for me is ``'\s\+$'``.

Reply:
Let's use `-E` instead

Reply:
Done.